src/test/unitTests/authentication.test.js

```diff
@@ -0,0 +1,56 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+// eslint-disable-next-line import/no-unresolved
+import { shuffleToken, generateToken, verifyToken } from '../../utils/authentication';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const payload = {
+  email: 'johndoe23@gmail.com',
+  password: '1234567'
+};
+
+describe('Tests for the authentication helper methods', () => {
+  describe('should generate a token', () => {
+    it('should return a shuffled token', () => {
+      const token = generateToken(payload);
+      expect(token).to.be.a('string');
+    });
```

Comment:
Add an assertion that the token being generated contains two `.`. i.e assert that when we call the `.split('.')` method on the token, we get an array of 3 elements